packages/manager/modules/pci/src/projects/project/kubernetes/components/update-policy-picker/update-policy-picker.constants.js

```diff
@@ -0,0 +1,7 @@
+// English only
+export const DOCUMENTATION_LINK =
+  'https://help.ovhcloud.com/csm/en-public-cloud-kubernetes-change-security-update?id=kb_article_view&sysparm_article=KB0049649';
```

Comment:
The documentation link should be specific to the subsidiary of the user's account.

Reply:
The documentation only exists in english, hence the comment on line 1